src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -30,16 +30,16 @@
     public List<OrderDto> getOrders() {
 
         List<OrderDto> list = new ArrayList<>();
-        list.add(new OrderDto(2L, "test 2"));
-        list.add(new OrderDto(3L, "test 3"));
+        list.add(new OrderDto());
+        list.add(new OrderDto());
```

Comment:
zwracanie 
```
list.add(new OrderDto(2L, "test 2"));
list.add(new OrderDto(3L, "test 3"));
```
jest ok. To I tak testowe (zamockowane) dane. Przy okazji mozesz sprawdzic serializacje danych (java => json)

Reply:
uwaga do `@CrossOrigin(origins = "*")`
- to generalnie jest ok, ale musimy ustawic corsy dla calej aplikacji. Zeby nie robic copy-paste w kazdym kontrolerze, lepiej jest uzyc *corsFilter*
https://spring.io/blog/2015/06/08/cors-support-in-spring-framework

przypomnij o tym prosze na spotkaniu

Reply:
Maćku dzieki za rewiev, klasy Service, Domain, Repository poprawiam - pytanie czy poprawiać kontroller bo ja go dotknąłem tylko w zakresie kompilacji kodu. Bo generalnie to późniejsze zadanie będzie i trzeba będzie go na nowo zakładam pisać....